client/Cargo.toml

```diff
@@ -39,6 +39,9 @@ base64 = "0.13.0"
 [dev-dependencies]
 iroha_core = { version = "=2.0.0-pre-rc.2", path = "../core", features = ["dev-telemetry", "telemetry"] }
 iroha_permissions_validators = { version = "=2.0.0-pre-rc.2", path = "../permissions_validators" }
+iroha = { path = "../cli", features = ["dev-telemetry", "telemetry"] }
+iroha_data_model = { version = "=2.0.0-pre-rc.2", path = "../data_model", features = ["warp", "cross_crate_testing"] }
```

Comment:
the same as `cli/Cargo.toml` comment. Features are probably lumped together

Reply:
Actually, let me quickly test it.